rel/files/ejabberd.cfg

```diff
@@ -662,18 +662,15 @@
                            % default: 600 seconds
                            % {resume_timeout, 600}
                           ]},
-%  {mod_muc_light, [
-%                   {host, "muclight.@HOST@"}
-%                  ]},
-  {mod_muc, [
```

Comment:
The market says MUC is obsolete, the XSF won't say it, or until MIX is widely deployed.

Indeed: MUC, MUC light, and MIX can co-exist.